2024-03-05 18:40:37,391 INFO    MainThread:19832 [wandb_setup.py:_flush():76] Current SDK version is 0.16.3
2024-03-05 18:40:37,393 INFO    MainThread:19832 [wandb_setup.py:_flush():76] Configure stats pid to 19832
2024-03-05 18:40:37,393 INFO    MainThread:19832 [wandb_setup.py:_flush():76] Loading settings from C:\Users\16979\.config\wandb\settings
2024-03-05 18:40:37,393 INFO    MainThread:19832 [wandb_setup.py:_flush():76] Loading settings from G:\My Drive\SI_630\Homeworks\Homework2 - Backup\wandb\settings
2024-03-05 18:40:37,393 INFO    MainThread:19832 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-03-05 18:40:37,394 INFO    MainThread:19832 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2024-03-05 18:40:37,394 INFO    MainThread:19832 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-03-05 18:40:37,397 INFO    MainThread:19832 [wandb_init.py:_log_setup():526] Logging user logs to G:\My Drive\SI_630\Homeworks\Homework2 - Backup\wandb\run-20240305_184037-yyqxozsf\logs\debug.log
2024-03-05 18:40:37,398 INFO    MainThread:19832 [wandb_init.py:_log_setup():527] Logging internal logs to G:\My Drive\SI_630\Homeworks\Homework2 - Backup\wandb\run-20240305_184037-yyqxozsf\logs\debug-internal.log
2024-03-05 18:40:37,399 INFO    MainThread:19832 [wandb_init.py:init():566] calling init triggers
2024-03-05 18:40:37,399 INFO    MainThread:19832 [wandb_init.py:init():573] wandb.init called with sweep_config: {}
config: {}
2024-03-05 18:40:37,399 INFO    MainThread:19832 [wandb_init.py:init():591] re-initializing run, found existing run on stack: dcfe52zc
2024-03-05 18:40:37,403 INFO    MainThread:19832 [wandb_run.py:_finish():1970] finishing run yanzhuo/attention_classifier/dcfe52zc
2024-03-05 18:40:37,403 INFO    MainThread:19832 [jupyter.py:save_history():445] not saving jupyter history
2024-03-05 18:40:37,404 INFO    MainThread:19832 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2024-03-05 18:40:37,404 INFO    MainThread:19832 [wandb_init.py:_jupyter_teardown():454] cleaning up jupyter logic
2024-03-05 18:40:37,404 INFO    MainThread:19832 [wandb_run.py:_atexit_cleanup():2210] got exitcode: 0
2024-03-05 18:40:37,407 INFO    MainThread:19832 [wandb_run.py:_restore():2193] restore
2024-03-05 18:40:37,408 INFO    MainThread:19832 [wandb_run.py:_restore():2199] restore done
2024-03-05 18:40:44,129 INFO    MainThread:19832 [wandb_run.py:_footer_history_summary_info():3866] rendering history
2024-03-05 18:40:44,129 INFO    MainThread:19832 [wandb_run.py:_footer_history_summary_info():3898] rendering summary
2024-03-05 18:40:44,155 INFO    MainThread:19832 [wandb_run.py:_footer_sync_info():3825] logging synced files
2024-03-05 18:40:44,550 INFO    MainThread:19832 [wandb_init.py:init():616] starting backend
2024-03-05 18:40:44,550 INFO    MainThread:19832 [wandb_init.py:init():620] setting up manager
2024-03-05 18:40:44,588 INFO    MainThread:19832 [backend.py:_multiprocessing_setup():105] multiprocessing start_methods=spawn, using: spawn
2024-03-05 18:40:44,588 INFO    MainThread:19832 [wandb_init.py:init():628] backend started and connected
2024-03-05 18:40:44,654 INFO    MainThread:19832 [wandb_run.py:_label_probe_notebook():1295] probe notebook
2024-03-05 18:40:44,658 INFO    MainThread:19832 [wandb_run.py:_label_probe_notebook():1305] Unable to probe notebook: 'charmap' codec can't decode byte 0x81 in position 3806: character maps to <undefined>
2024-03-05 18:40:44,658 INFO    MainThread:19832 [wandb_init.py:init():720] updated telemetry
2024-03-05 18:40:44,748 INFO    MainThread:19832 [wandb_init.py:init():753] communicating run to backend with 90.0 second timeout
2024-03-05 18:40:45,810 INFO    MainThread:19832 [wandb_run.py:_on_init():2262] communicating current version
2024-03-05 18:40:46,290 INFO    MainThread:19832 [wandb_run.py:_on_init():2271] got version response 
2024-03-05 18:40:46,291 INFO    MainThread:19832 [wandb_init.py:init():804] starting run threads in backend
2024-03-05 18:40:46,586 INFO    MainThread:19832 [wandb_run.py:_console_start():2241] atexit reg
2024-03-05 18:40:46,587 INFO    MainThread:19832 [wandb_run.py:_redirect():2096] redirect: wrap_raw
2024-03-05 18:40:46,587 INFO    MainThread:19832 [wandb_run.py:_redirect():2161] Wrapping output streams.
2024-03-05 18:40:46,588 INFO    MainThread:19832 [wandb_run.py:_redirect():2186] Redirects installed.
2024-03-05 18:40:46,598 INFO    MainThread:19832 [wandb_init.py:init():847] run started, returning control to user process
2024-03-05 18:41:42,463 WARNING MsgRouterThr:19832 [router.py:message_loop():77] message_loop has been closed
